core/controllers/editor.py

```diff
@@ -411,7 +411,7 @@ def put(self, exploration_id: str) -> None:
                     'Sorry, we could not find the specified user.')
             if new_member_role is None:
                 raise self.InvalidInputException(
-                    'No role given for new member.')
+                    'Please provide a role to a new member for exploration.')
```

Comment:
Please provide a role for the new member of the exploration.

Reply:
Done!